tests/sqlparser_postgres.rs

```diff
@@ -3500,3 +3500,69 @@ fn parse_join_constraint_unnest_alias() {
         }]
     );
 }
+
+#[test]
+fn test_complex_postgres_insert_with_alias() {
+    let sql1 = "WITH existing AS (SELECT test_table.id FROM test_tables AS test_table WHERE (a = 12) AND (b = 34)), inserted AS (INSERT INTO test_tables AS test_table (id, a, b, c) VALUES (DEFAULT, 56, 78, 90) ON CONFLICT(a, b) DO UPDATE SET c = EXCLUDED.c WHERE (test_table.c <> EXCLUDED.c)) SELECT c FROM existing";
+
+    pg().verified_stmt(sql1);
+
+    let sql2 = "INSERT INTO test_tables AS test_table (id, a) VALUES (DEFAULT, 123)";
```

Comment:
Could you please:
1. Make the `sql2` a second test (e.g. `test_simple_postgres_insert_with_alias`)
2. Add a test for a quoted identifier, something like  `INSERT INTO test_tables AS "Test_Table" (id, a) VALUES (DEFAULT, 123)`, perhaps